<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteManager</class>
 <widget class="QWidget" name="PaletteManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palette Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current palette</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="CurrentList"/>
     </item>
    </layout>
   </item>
   <item row="0" column="4" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddFloss">
       <property name="text">
        <string notr="true">&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveFloss">
       <property name="text">
        <string notr="true">&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="6" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Current scheme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CurrentScheme">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="6" colspan="2">
    <widget class="QListWidget" name="ColorList"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Symbol Library</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="KComboBox" name="SymbolLibrary"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Stitch strands</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="StitchStrands">
     <item>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Symbol</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="StitchSymbol">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>158</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="7" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="NewFloss">
       <property name="text">
        <string>New Floss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="ClearUnused">
       <property name="text">
        <string>Clear Unused Flosses</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Calibrate">
       <property name="text">
        <string>Calibrate Flosses</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Backstitch strands</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="BackstitchStrands">
     <item>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Symbol</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="KComboBox" name="BackstitchSymbol">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string comment="A solid line">Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="A dashed line">Dashed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="A dotted line">Dotted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash Dotted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash Dash Dotted</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
